webapp/src/game/game.controller.ts

```diff
@@ -81,11 +96,14 @@ export class GameController {
       state,
     );
 
+    response.cookie("gameId", foundGame.id);
+
     state = gameState;
 
     if (!allowedMove) {
       await this.gameService.updateBoard(foundGame.id, board);
       await this.gameService.updateGameState(foundGame.id, state);
+      this.eventsService.emit({ data: new Date().toISOString() }, sseId);
```

Comment:
Create a function instead that does :
```ts
      await this.gameService.updateBoard(foundGame.id, board);
      await this.gameService.updateGameState(foundGame.id, state);
      this.eventsService.emit({ data: new Date().toISOString() }, sseId);
      return response.redirect(`/game/${JSON.stringify(foundGame.id)}`);
```

You begin to have a lot of duplicated code